src/slider/styled-components.js

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+import {startThumbIcon, endThumbIcon, singleThumbIcon} from './icons';
+
+export const Root = styled('div', props => {
+  return {
+    paddingTop: '1px',
+    position: 'relative',
+  };
+});
+
+export const Axis = styled('div', props => {
+  const {$theme} = props;
+  const {colors, borders, sizing} = $theme;
+  return {
+    position: 'relative',
+    marginTop: sizing.scale900,
+    marginBottom: sizing.scale900,
+    marginRight: sizing.scale400,
+    marginLeft: sizing.scale400,
+    borderRadius: $theme.borders.useRoundedCorners ? borders.radius100 : '0px',
+    background: colors.mono400,
+    height: sizing.scale100,
+  };
+});
+
+export const AxisRange = styled('div', props => {
+  const {$max, $min, $index, $isRange, $value, $theme} = props;
+  const {colors, borders, sizing} = $theme;
+  const fillColor = colors.primary400;
+  const emptyColor = colors.mono400;
+  let barColor;
+  if ($isRange) {
+    barColor = $index % 2 === 0 ? emptyColor : fillColor;
+  } else {
+    barColor = $index % 2 === 0 ? fillColor : emptyColor;
+  }
+  const value =
+    $isRange && $index ? $value[$index] - $value[$index - 1] : $value[$index];
+  const $successValue = $max - $min;
```

Comment:
What is `successValue`?

Reply:
renamed to `totalRangeValue`, previous was confused from other code